taxonomy/management/commands/refresh_course_skills.py

```diff
@@ -18,10 +18,12 @@
 
 class Command(BaseCommand):
     """
-        Example usage:
-            $ ./manage.py refresh_course_skills --course 'Course1_uuid' --course 'Course2_uuid' --commit
-            $ ./manage.py refresh_course_skills --args-from-database
-        """
+    Command to referesh skills associated with the courses.
+
+    Example usage:
+        $ ./manage.py refresh_course_skills --course 'Course1_uuid' --course 'Course2_uuid' --commit
+        $ ./manage.py refresh_course_skills --args-from-database
```

Comment:
Do we need to explain little bit about what is `args-from-database`

Reply:
@irfanuddinahmad can you explain the purpose of `args-from-database` to add it in the documentation?